api/client/client_test.go

```diff
@@ -97,7 +97,7 @@ func TestNewMutualTLSServer(t *testing.T) {
 	if err == nil {
 		t.Fatalf("expected error with sign function")
 	}
-	if !strings.Contains(err.Error(), "Post https://nohost:8888/api/v1/cfssl/sign: dial tcp: lookup nohost") {
+	if !(strings.Contains(err.Error(), "Post")) && !(strings.Contains(err.Error(), "https://nohost:8888/api/v1/cfssl/sign")) && !(strings.Contains(err.Error(), "dial tcp: lookup nohost: no such host")) {
```

Comment:
Looking at the change that was done in 1.14, the adding escaped quotes around the URL, this would work too
```
!strings.Contains(err.Error(), "Post \\\"https://nohost:8888/api/v1/cfssl/sign\\\": dial tcp: lookup nohost")
```

Reply:
Yes! That works too! But only for go1.14, which means that it will fail for go1.13 or less..

Reply:
Oh good point, sounds good.